src/test/java/nextstep/subway/path/domain/DefaultFarePolicyTest.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.subway.path.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DefaultFarePolicyTest {
+
+    @DisplayName("기본요금 계산")
+    @ParameterizedTest
+    @ValueSource(longs = {0L, 9L, 10L, 11L, 12L, 50L, 100L})
+    void 기본요금_계산(Long distance) {
+        //given
+        DefaultFarePolicy defaultFarePolicy = new DefaultFarePolicy();
+        // when
+        Long fare = defaultFarePolicy.applyPolicy(distance);
+        // then
+        assertThat(fare).isEqualTo(defaultFarePolicy.getDEFAULT_FARE());
```

Comment:
상수라면 아래와 같이 접근해보는 건 어떨까요?
```suggestion
        assertThat(fare).isEqualTo(DefaultFarePolicy.DEFAULT_FARE);
```